app/src/components/RSSArticle.js

```diff
@@ -154,12 +155,14 @@ RSSArticle.defaultProps = {
 RSSArticle.propTypes = {
 	_id: PropTypes.string,
 	commentUrl: PropTypes.string,
-	enclosures: PropTypes.arrayOf(PropTypes.shape({
-		_id: PropTypes.string,
-		url: PropTypes.string,
-		type: PropTypes.string,
-		length: PropTypes.string,
-	})),
+	enclosures: PropTypes.arrayOf(
+		PropTypes.shape({
+			_id: PropTypes.string,
+			url: PropTypes.string,
+			type: PropTypes.string,
+			length: PropTypes.string,
+		}),
+	),
```

Comment:
enclosures can also have audio content, we should check that the type contains "video" or "youtube"

Reply:
I didn't know how to handle that in react.